.pre-commit-config.yaml

```
@@ -21,28 +21,28 @@ repos:
     hooks:
     -   id: autopep8
 -   repo: https://github.com/PyCQA/flake8
-    rev: 7.0.0
+    rev: 7.3.0
     hooks:
     -   id: flake8
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.8.0
+    rev: v1.16.1
     hooks:
     -   id: mypy
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)